ferrocene/ci/docker-images/ubuntu-20/Dockerfile

```diff
@@ -63,18 +63,18 @@ RUN apt-get update \
 # Grab the newer QEMU from Ubuntu 22.04
 # This requires the binfmt-support package, which is normally a dependency of
 # qemu-user-static.
-COPY --from=jammy /usr/bin/qemu-aarch64-static /usr/bin/qemu-aarch64-static
-COPY --from=jammy /usr/libexec/qemu-binfmt/aarch64-binfmt-P /usr/libexec/qemu-binfmt/aarch64-binfmt-P
-COPY --from=jammy /usr/share/binfmts/qemu-aarch64 /usr/share/binfmts/qemu-aarch64
+COPY --from=noble /usr/bin/qemu-aarch64-static /usr/bin/qemu-aarch64-static
+COPY --from=noble /usr/libexec/qemu-binfmt/aarch64-binfmt-P /usr/libexec/qemu-binfmt/aarch64-binfmt-P
+# COPY --from=noble /usr/share/binfmts/qemu-aarch64 /usr/share/binfmts/qemu-aarch64
```

Comment:
Why is this commented?

Reply:
The file doesn't exist anymore, seems I need to hunt for it.